### ISSUE TITLE
pyparsing.exceptions.ParseException: Expected '}', found '=' 

### ISSUE THREAD

**Issue by lemonsstyle** [mabaorui/NeuralPull-Pytorch#5]
Hello, I want to test with your code, but it directly reports an error before it reads my ply file:

pyparsing.exceptions.ParseException: Expected '}', found '='  (at char 463), (line:34, col:14)
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "run.py", line 190, in <module>
    runner = Runner(args, args.conf, args.mode)
  File "run.py", line 32, in __init__
    self.conf = ConfigFactory.parse_string(conf_text)
  File "/home/west/.conda/envs/effi/lib/python3.8/site-packages/pyhocon/config_parser.py", line 192, in parse_string
    return ConfigParser().parse(content, basedir, resolve, unresolved_value)
  File "/home/west/.conda/envs/effi/lib/python3.8/site-packages/pyhocon/config_parser.py", line 455, in parse
    config = config_expr.parseString(content, parseAll=True)[0]
  File "/home/west/.local/lib/python3.8/site-packages/pyparsing/core.py", line 1141, in parse_string
    raise exc.with_traceback(None)
pyparsing.exceptions.ParseSyntaxException: Expected '}', found '='  (at char 463), (line:34, col:14)

I didn't understand the serial number of this column.My device is ubuntu 20.04, torch=1.12.1, cuda=11.4, RTX 3090, python=3.8.0.

**Comment by junshengzhou** [comment]
Hi,
The reason is the version of pyhocon, the newest version no longer support some characteristics. 
Try using pyhocon==0.3.57.

**Comment by lemonsstyle** [comment]
> Hi, The reason is the version of pyhocon, the newest version no longer support some characteristics. Try using pyhocon==0.3.57.

Thanks!  Your method is very useful. After modification, I tested my dataset, but it contains millions of points. This time my error is consistent with another friend's question. I modified "QUERY_EACH=1000000/POINT_NUM_GT" according to your reply, and the error still exists. It seems that my data is too large  : (  , so I try to keep only a small part of the point cloud for reconstruction.  I am reading your article and  waiting for the effect of this time (the reconstruction speed is very slow).
